src/Native/System.Security.Cryptography.Native/configure.cmake

```diff
@@ -7,8 +7,6 @@ set(CMAKE_REQUIRED_INCLUDES ${OPENSSL_INCLUDE_DIR})
 check_library_exists(${OPENSSL_SSL_LIBRARY} "TLSv1_1_method" "" HAVE_TLS_V1_1)
```

Comment:
Do we still need these checks?  Is "CIPHER_SPLIT_ALGORITHMS" the only conditional compilation we can eliminate?